pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -943,4 +943,6 @@ self: super: {
   # Add support for https://github.com/haskell-hvr/multi-ghc-travis.
   multi-ghc-travis = self.callPackage ../tools/haskell/multi-ghc-travis { ShellCheck = self.ShellCheck_0_4_6; };
 
+  shelly = dontCheck super.shelly;
```

Comment:
Has the issue that causes the tests to fail been reported to upstream? If it has, then please add a link to the upstream ticket before that line so that other people can figure out what is going on.